cookbooks/ambari/recipes/ambari_server_setup.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Cookbook Name:: ambari
+# Recipe:: ambari_server_setup
+#
+# Copyright (c) 2016 Artem Ervits, All Rights Reserved.
+
+
+# /etc/ambari-server/conf/ambari.properties
+template "#{node['ambari']['ambari_server_conf_dir']}ambari.properties" do
```

Comment:
Please use [`File.join`](http://ruby-doc.org/core-2.2.0/File.html#method-c-join) so if one does not leave a trailing `/` on their directory things will not mysteriously fail.